crypto/asn1/a_time.c

```diff
@@ -124,14 +124,14 @@ int asn1_time_to_tm(struct tm *tm, const ASN1_TIME *d)
             i++;
             break;
         }
-        if (!isdigit(a[o]))
+        if (!ossl_isdigit(a[o]))
             goto err;
         n = a[o] - '0';
```

Comment:
Use of `'0'`  is a potential problem with EBCDIC in ASN.1 code. Specifically here, a[o] is a digit in the ASCII numeric space 0x30~0x39, and when compiling for EBCDIC, `'0'` has a value of 0xF0. Should we have a ossl_toascii() macro or something (proper translation in EBCDIC, no-op in ASCII), that allows us to still use character constants, but safely in the ASN.1 code?